config/_default/params.yaml

```diff
@@ -5,12 +5,10 @@ version_menu: "Releases"
 # The version number for the version of the docs represented in this doc set.
 # Used in the "version-banner" partial to display a version number for the
 # current doc set.
-version: "v0.10.0"
-
-archived_version: true
+version: latest
 url_latest_version: https://projectcapsule.dev
+archived_version: false
 versions:
 - version: v0.10.0
-  url: https://projectcapsule.dev
-#- version: v0.10.0
-#  url: https://release-0-10-0.projectcapsule.dev
\ No newline at end of file
+  url: https://release-0-10-0.projectcapsule.dev
```

Comment:
Maybe Something like this would be better (just looking at how kyverno does it (https://github.com/kyverno/website/blob/main/config/_default/params.toml)


```yaml
version: latest
url_latest_version: "https://latest.projectcapsule.dev"
versions:
- version: "latest"
  url: "https://latest.projectcapsule.dev"
- version: "v0.10.0"
  url: https://release-0-10-0.projectcapsule.dev



```